EnglishTranslation-master/QUEST_JOBSTEP.tsv

```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
+QUEST_JOBSTEP_20150803_003100	If you will be able to pick out this kind of object, you will become a stronger Necromancer.{nl}For now, I'll reward you if you can find one.
+QUEST_JOBSTEP_20150803_003101	Ah, you came at the right time. {nl}I am investigating Crystal Mine, but it will be dangerous if we leave it as is, so I want to take care of it quickly.{nl}
+QUEST_JOBSTEP_20150803_003102	If you could help me, I think the investigation would be done sooner.{nl}And it will be also a great chance to test your skills as an Alchemist.
+QUEST_JOBSTEP_20150803_003103	I am glad that Vaidotas asked me for a favor.{nl}Return with this.
+QUEST_JOBSTEP_20150803_003104	Good.{nl}I need the orb that records the memories of the past from the Chronomancer Master.{nl}He is the head of our Winterspoon Family, so I've requested it from him in advance. Tell him that I've sent you.
+QUEST_JOBSTEP_20150803_003105	Lucid...Actually have you met the Chronomancer Master well?
+QUEST_JOBSTEP_20150803_003106	The orb reads the past memories of its owner and records them.{nl}Completely explore District 1 and 2 in the Crystal Mine with this.{nl}
+QUEST_JOBSTEP_20150803_003107	When you use this orb after exploring completely, it will read the memories of the possessor and record the past.{nl}If you had explored the regions in the past, then record it into the orb right away.{nl}
+QUEST_JOBSTEP_20150803_003108	Well done. {nl}It will take some time to investigate this, so in the meantime, I will teach you new alchemy techniques.
 QUEST_JOBSTEP_20150803_003109	Eternal Amjinas [Sorcerer Advancement]
 QUEST_JOBSTEP_20150803_003110	Tell him that it will be better to listen directly
 QUEST_JOBSTEP_20150803_003111	I don't have guts for it
 QUEST_JOBSTEP_20150803_003112	Received enough balm. Restore the Statue of Goddess Vakarine in Fedimian and Statue of Goddess Ausrine in Klaipeda.
 QUEST_JOBSTEP_20150803_003113	The Sorcerer Master wants to hear more from Amjinases. You can meet the Dievdirbys Master in West Siauliai Woods, the Necromancer Master in Mullers Passage, and the Chronomancer Master in Stele Road.
 QUEST_JOBSTEP_20150803_003114	You've heard about eternal lives from Amjinases. Return to the Sorcerer Master.
-QUEST_JOBSTEP_20150803_003115	Obtain the keepsakes from which you could feel strong regrets.
-QUEST_JOBSTEP_20150803_003116	The Necromancer Master wants to have the keepsake with a stronger grudge. Obtain the keepsakes that you can feel strong regrets from Videntis Shrine.
-QUEST_JOBSTEP_20150803_003117	You've collected the keepsakes with strong regrets. Return to the Necromancer Master.
+QUEST_JOBSTEP_20150803_003115	Obtain the keepsakes that emit a strong sense of regret.
+QUEST_JOBSTEP_20150803_003116	The Necromancer Master wants keepsakes with a stronger grudge. Obtain the menacing keepsakes from Videntis Shrine.
+QUEST_JOBSTEP_20150803_003117	You've collected the keepsakes, of which have a heavy aura about them. Return to the Necromancer Master.
```

Comment:
maybe just have it as `You've collected the keepsakes that have a heavy aura about them.` ?